lti_consumer/models.py

```diff
@@ -227,17 +227,14 @@ class LtiConfiguration(models.Model):
                   'grades.'
     )
 
-    # Empty variable that'll hold the block once it's retrieved
-    # from the modulestore or preloaded
-    _block = None
-
     def clean(self):
         if self.config_store == self.CONFIG_ON_XBLOCK and self.location is None:
             raise ValidationError({
                 "config_store": _("LTI Configuration stores on XBlock needs a block location set."),
             })
         if self.version == self.LTI_1P3 and self.config_store == self.CONFIG_ON_DB:
-            if not database_config_enabled(self.block.location.course_key):
+            block = compat.load_enough_xblock(self.location)
```

Comment:
@ashultz0 so this line will cause an exception if this is run outside of edx-platform like in exams. I think maybe a try/catch would be enough to fix it since we don't care about the xmodule import in compat failing outside platform.

Reply:
@Zacharis278 actually I'm realizing that the only thing it does with the block here is load it by the location and then check the block's location's course key - I should be able to just check the location's course key without loading the block right?